src/LinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -175,7 +175,7 @@ static std::optional<bool> HasFramePointerSetFromDwarfSection(
 std::optional<bool> LibunwindstackUnwinderImpl::HasFramePointerSet(uint64_t instruction_pointer,
                                                                    pid_t pid,
                                                                    unwindstack::Maps* maps) {
-  unwindstack::MapInfo* map_info = maps->Find(instruction_pointer);
+  unwindstack::MapInfo* map_info = maps->Find(instruction_pointer).get();
```

Comment:
Same here

Reply:
Done.

Reply:
Maybe you forgot to push? Ownership is not shared locally...

Reply:
Oof, thanks, I got a bit confused... Unfortunately below a bare pointer needs to be passed back to libunwindstack, but let's do as well as we can.